app/models/repository.rb

```diff
@@ -1,3 +1,7 @@
 class Repository < KalibroClient::Entities::Processor::Repository
   include KalibroRecord
+
+  def self.latest(count=1)
+    all.sort { |one, another| another.id <=> one.id }.first(count)
```

Comment:
Maybe we can measure the time it takes on both cases. If the difference is really of two or three orders of magnitude, I think we should do it even if readability gets worse.

The reason for that is we do not want that our pages take more than a few hundreds of milliseconds to load. Specially our home page, even on a few cases (when cache needs to be updated). There's been a lot of research concluding that users will not return to your site if the pages take too long to load (usually that means at most 1 second).